fix.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -x -e
 
+xmlstarlet ed -L \
+	-d '//_:source-position' \
```

Comment:
Gir support `source-position`, if we missed some places, then better fix it in gir

Reply:
I must have needed a new version of gir, done.